Documentation/Contributing/documentation.md

```diff
@@ -36,7 +36,7 @@ When previewing, now you can navigate your browser to [http://127.0.0.1:8000/](h
     Please make sure that your Python binary path is included in your `PATH`.
 
 ## Running helm-docs
-[helm-docs](https://github.com/norwoodj/helm-docs) is a tool that generates the documentation for a helm chart automatically. If there are changes in the helm chart, you need to run helm-docs manually, and check in the resulting autogenerated md files at the path `/Documentation/Helm-Charts`
+[helm-docs](https://github.com/norwoodj/helm-docs) is a tool that generates the documentation for a helm chart automatically. If there are changes in the helm chart, you need to run helm-docs manually, and check in the resulting autogenerated md files at the path `/Documentation/Helm-Charts`. We have made running helm-docs very easy for you by creating a `make` target for it: you only need to run `make helm-docs`. an additional `make` target exists to help you check if there are  uncommitted generated helm chart docs: just run `make check-helm-docs`  
```

Comment:
Rook has a new initiative to make docs more imperative (notably avoiding "we" and "you" in docs), so this is a good time to update this paragraph.

```suggestion
[helm-docs](https://github.com/norwoodj/helm-docs) is a tool that generates the documentation for a helm chart automatically. If there are changes in the helm chart, run `make helm-docs`, and check in the resulting autogenerated markdown files at the path `/Documentation/Helm-Charts`.
```

After thinking about this, if I imagine myself to be a new contributor, I think I would be confused how running `make helm-docs` is improved by also running `make check-helm-docs`. I think it might end up being less confusing new contributors to have this target be something that is not officially documented.

Improvements for new contributors is certainly valuable, and reducing the barrier to entry is worthwhile. In service of that and in a related vein as this PR, something that I think would be most impactful and useful would be to have a make target that runs all auto-generation targets including docs, helm-docs, crds, and any I forgot. The only exception I would make would be `make codegen` which takes too long to run.

Reply:
@BlaineEXE  - do I get it right that you are requesting to remove the new mention of the new make target from this section altogether? I can also try to make the wording clearer and avoid using "we" and "you" ...

Reply:
@obnoxxx  wrote:
> @BlaineEXE - do I get it right that you are requesting to remove the new mention of the new make target from this section altogether?

>Alternatively, I can also try to make the wording clearer and avoid using "we" and "you" ...

I have done that and updated the PR

Reply:
> @BlaineEXE - do I get it right that you are requesting to remove the new mention of the new make target from this section altogether? I can also try to make the wording clearer and avoid using "we" and "you" ...

Yes. When I read this while imagining standing in the shoes of a new contributor, I think I would feel overwhelmed by the length of the text and confused about what make targets to run. Short and sweet is best, and I've especially noticed this to be true for users/contributors who are in Asia and don't have the same mastery of English.